tests/components/cloud/test_client.py

```diff
@@ -381,3 +385,50 @@ async def test_cloud_connection_info(hass: HomeAssistant) -> None:
         "version": HA_VERSION,
         "instance_id": "12345678901234567890",
     }
+
+
+@pytest.mark.parametrize(
+    "translation_key",
+    sorted(VALID_REPAIR_TRANSLATION_KEYS),
+)
+async def test_async_create_repair_issue_known(
+    hass: HomeAssistant,
+    issue_registry: IssueRegistry,
+    translation_key: str,
+) -> None:
+    """Test create repair issue for known repairs."""
+    with patch("hass_nabucasa.Cloud.initialize"):
+        assert await async_setup_component(hass, "cloud", {"cloud": {}})
+    cloud = hass.data["cloud"]
```

Comment:
There's a `cloud` fixture now that we may be able to use.

Reply:
Where is that used?
>  AttributeError: Mock object has no attribute 'client'

Reply:
Here's an example:
https://github.com/home-assistant/core/blob/bb6f78dcc49a9a6afad56173d5ea01d5ad236b4e/tests/components/cloud/test_http_api.py#L957-L982

Reply:
The integration still needs to be setup via `async_setup_component`.

Reply:
Okey, so it still needs a setup mock.. check.